### PR TITLE
[tests-only]tests: add tests for removing share permission in group share

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
https://owncloud.dev/libre-graph-api/#/drives.permissions/DeletePermission
Scenarios added:
1. Scenario Outline: user removes access to a resource in a group share
2. Scenario Outline: user removes access to a resource inside of a project space in group share

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/8008

## Motivation and Context
To provide test coverage for `/v1beta1/drives/{drive-id}/items/{item-id}/permissions/{perm-id}` 

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/SharingNgContext.php [hunk 1]
```diff
@@ -296,27 +297,37 @@ public function userUpdatesLastPublicLinkShareUsingTheGraphApiWith(string $user,
...
+			$sharer,
+			$userIdOfSharee,
+			$this->featureContext->getPasswordForUser($sharer),
 			$spaceId,
-			$itemId
+			$itemId,
+			$shareType
 		);
 		$this->featureContext->setResponse(
 			GraphHelper::deleteSharePermission(
```
Comment (SagarGi) [inline tests/acceptance/features/bootstrap/SharingNgContext.php:326]:
```suggestion
			GraphHelper::removeSharePermission(
```
May be change also the function name.

**Review by SagarGi — COMMENTED** [review]
other LGTM :+1:

File: tests/TestHelpers/GraphHelper.php [hunk 2]
```diff
@@ -1543,11 +1543,12 @@ public static function getPermissionsList(
 	/**
 	 * @param string $baseUrl
 	 * @param string $xRequestId
-	 * @param string $user
-	 * @param string $userIdOfShareeUser
+	 * @param string $sharer
+	 * @param string $userIdOfShareeUser this is a uuidv4 of sharer
```
Comment (PrajwolAmatya) [inline tests/TestHelpers/GraphHelper.php:1547]: then, also need to update the parameter name as well from sharee to sharer

Comment (SwikritiT) [reply tests/TestHelpers/GraphHelper.php:1547]: that's actually of sharee user

**Review by saw-jan — APPROVED** [review]
:eyes: good :+1:

**Review by SagarGi — APPROVED** [review]

**Pull request merged**